### PR TITLE
Update PyYAML to 5.1

### LINKED ISSUE #23: Update pyyaml when 4.2 release is available
## Description

We got a GitHub security alert regarding a CVE in pyyaml: [CVE-2017-18342](https://nvd.nist.gov/vuln/detail/CVE-2017-18342). The fix, [merged in the pyyaml repo](https://github.com/yaml/pyyaml/pull/74), renames `safe_load` to `load` and `load` to `danger_load`. We're not using `safe_load` for this project, which we should do in the beta release if pyyaml 4.2 is not ready. This is acceptable for now because:

* The only YAML load call in this project is for the [proxy config](https://github.com/freedomofpress/securedrop-proxy/blob/df6490d28c67f56584968fedf0a506f333785eed/securedrop_proxy/config.py#L18), which is generated via the workstation provisioning code running in dom0. 
* This project is in alpha stage and is not used in any production environments.

We shouldn't update to latest yet due to the fact that the only 4.2* versions available on PyPI are marked as pre-release - see https://github.com/yaml/pyyaml/issues/193 for the full details. 

tl;dr We should wait until pyyaml 4.2 is officially released and then update.

### PR DESCRIPTION
**Pull request by rmol**
This change updates the PyYAML requirement to 5.1.

Fixes #23.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="securedrop-proxy",
-    version="0.1.2",
+    version="0.1.3",
```
Comment (redshiftzero) [inline setup.py]: the other place we'll want to bump the version is in the proxy `Response` here: https://github.com/freedomofpress/securedrop-proxy/blob/master/securedrop_proxy/proxy.py#L21

**Comment by redshiftzero** [comment]
hey @kushaldas - want to do the diff/changelog review for these dependency updates?

**Comment by redshiftzero** [comment]
e56f799 lgtm! I just enabled builds from forks in Circle CI so that tests will run on next push

**Comment by redshiftzero** [comment]
To the diff/changelog reviewer: we reviewed pyyaml updates over in https://github.com/freedomofpress/securedrop-sdk/pull/72 so the reviewer here just needs to verify the hashes are the same

**Comment by kushaldas** [comment]
@redshiftzero as we discussed yesterday, we will review the source code diffs of the `pyyaml` while building/syncing wheels step. For this PR, we will not need that.

**Comment by kushaldas** [comment]
@rmol @redshiftzero the update to PyYAML 5.1 is approved. The diff looks okay to me.

https://github.com/freedomofpress/securedrop-debian-packaging/wiki/PyYAML-5.1

**Pull request merged**